### PR TITLE
Prettier 3.0

### PR DESCRIPTION
**Pull request by turadg**
## Description

https://prettier.io/blog/2023/07/05/3.0.0.html

Also has a 'yarn rebuild' of cosmic-proto that was affected by https://github.com/Agoric/agoric-sdk/pull/7822 

uses alpha version of `eslint-plugin-prettier` due to https://github.com/prettier/eslint-plugin-prettier/issues/562

### Testing Considerations

CI

### CODE REVIEW

File: packages/agoric-cli/src/commands/psm.js [hunk 1]
```diff
@@ -151,15 +151,17 @@ export const makePsmCommand = logger => {
...
-    });
+    .action(
+      async function (
+        /** @type {Parameters<typeof Offers.psm.swap>[2]} */ opts,
+      ) {
+        console.warn('running with options', opts);
+        const { agoricNames, lookupPsmInstance } = await rpcTools();
+        const instance = await lookupPsmInstance(opts.pair);
+        const offer = Offers.psm.swap(agoricNames, instance, opts);
+        outputExecuteOfferAction(offer);
```
Comment (dckc) [inline packages/agoric-cli/src/commands/psm.js:162]: sigh... indenting the whole body of the function is mildly annoying. If I saw this while writing code, I might shift things around.

File: packages/cosmic-proto/dist/agoric/swingset/msgs.d.ts [hunk 2]
```diff
@@ -506,15 +506,15 @@ interface Rpc {
     data: Uint8Array,
   ): Promise<Uint8Array>;
 }
-declare type Builtin =
+type Builtin =
```
Comment (dckc) [inline packages/cosmic-proto/dist/agoric/swingset/msgs.d.ts:509]: ugh. this code is generated by some protobuf tool. next time we run that tool (`yarn generate`) it's going to say `declare` again, IIUC. I wonder if this is a known issue upstream in the protobuf tool.

cc @kriskowal @mhofman

File: packages/cosmic-proto/dist/agoric/swingset/msgs.js [hunk 3]
```diff
@@ -64,10 +64,10 @@ export const MsgDeliverInbound = {
   fromJSON(object) {
     return {
       messages: Array.isArray(object?.messages)
-        ? object.messages.map((e) => String(e))
+        ? object.messages.map(e => String(e))
```
Comment (dckc) [inline packages/cosmic-proto/dist/agoric/swingset/msgs.js:67]: again, this is generated code. Any clues on why our lint tools didn't complain about it when it was originally merged?

**Review by dckc — APPROVED** [review]
I'm a bit concerned about the generated `declare` things, but I suppose the tools will catch them next time too.

File: packages/cosmic-proto/dist/agoric/swingset/msgs.d.ts (hunk 2 above)
Comment (turadg) [reply packages/cosmic-proto/dist/agoric/swingset/msgs.d.ts:509]: this was generated by `yarn rebuild` which runs `yarn generate`. I put this into a separate commit, before changing Prettier: 87a2df6b037c44efdd42ad35a1d2e347e4efca22

File: packages/cosmic-proto/dist/agoric/swingset/msgs.js (hunk 3 above)
Comment (turadg) [reply packages/cosmic-proto/dist/agoric/swingset/msgs.js:67]: this is the generated code after being prettiered (`yarn rebuild` runs prettier). The change in b46c46 is to use the same Prettier config for this package as for agoric-sdk, i.e. `"arrowParens": "avoid"`

**Review by mhofman — DISMISSED** [review]
Linting tools should not touch generated code

File: packages/cosmic-proto/dist/agoric/swingset/msgs.js (hunk 3 above)
Comment (mhofman) [reply packages/cosmic-proto/dist/agoric/swingset/msgs.js:67]: I'm confused, are you saying the codegen steps already re-format generated code?

Comment (turadg) [reply packages/cosmic-proto/dist/agoric/swingset/msgs.js:67]: Yes. https://github.com/Agoric/agoric-sdk/blob/1303537a9b1857100515533428e26193f975453b/packages/cosmic-proto/package.json#L38

**Pull request merged**